task_definitions/staging/task_definition.json

```diff
@@ -0,0 +1,194 @@
+{
+  "taskDefinitionArn": "arn:aws:ecs:us-east-2:693260174422:task-definition/ette:1",
+  "containerDefinitions": [
+    {
+      "name": "kafka",
+      "image": "confluentinc/cp-kafka:latest",
+      "cpu": 0,
+      "links": [],
+      "portMappings": [
+        {
+          "containerPort": 29090,
+          "hostPort": 0,
+          "protocol": "tcp"
+        }
+      ],
+      "essential": true,
+      "entryPoint": [],
+      "command": [],
+      "environment": [
+        {
+          "name": "KAFKA_BROKER_ID",
+          "value": "1"
+        },
+        {
+          "name": "KAFKA_ADVERTISED_LISTENERS",
+          "value": "PLAINTEXT://kafka:9092,PLAINTEXT_HOST://localhost:29092"
+        },
+        {
+          "name": "KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR",
+          "value": "1"
+        },
+        {
+          "name": "KAFKA_INTER_BROKER_LISTENER_NAME",
+          "value": "PLAINTEXT"
+        },
+        {
+          "name": "KAFKA_ZOOKEEPER_CONNECT",
+          "value": "zookeeper:2181"
+        },
+        {
+          "name": "KAFKA_LISTENER_SECURITY_PROTOCOL_MAP",
+          "value": "PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT"
+        }
+      ],
+      "environmentFiles": [],
+      "mountPoints": [],
+      "volumesFrom": [],
+      "secrets": [],
+      "dnsServers": [],
+      "dnsSearchDomains": [],
+      "extraHosts": [],
+      "dockerSecurityOptions": [],
+      "dockerLabels": {},
+      "ulimits": [],
+      "systemControls": []
+    },
+    {
+      "name": "cache",
+      "image": "redis:6.0.6-alpine",
+      "cpu": 0,
+      "links": [],
+      "portMappings": [
+        {
+          "containerPort": 6378,
+          "hostPort": 0,
+          "protocol": "tcp"
+        }
+      ],
+      "essential": true,
+      "entryPoint": [],
+      "command": ["redis-server --save 20 1 --loglevel warning"],
+      "environment": [],
+      "environmentFiles": [],
+      "mountPoints": [],
+      "volumesFrom": [],
+      "secrets": [],
+      "dnsServers": [],
+      "dnsSearchDomains": [],
+      "extraHosts": [],
+      "dockerSecurityOptions": [],
+      "dockerLabels": {},
+      "ulimits": [],
+      "systemControls": []
+    },
+    {
+      "name": "db",
+      "image": "postgres:14.1-alpine",
+      "cpu": 0,
+      "links": [],
+      "portMappings": [
+        {
+          "containerPort": 5432,
+          "hostPort": 0,
+          "protocol": "tcp"
+        }
+      ],
+      "essential": true,
+      "entryPoint": [],
+      "command": [],
+      "environment": [
+        {
+          "name": "POSTGRES_USER",
+          "value": "ette_db"
+        },
+        {
+          "name": "POSTGRES_PASSWORD",
+          "value": "ette_db"
```

Comment:
For both postgres username and password use secrets... valueFrom option do not use plain strings for passwords, secrets